src/Microsoft.Framework.PackageManager/Restore/RestoreCommand.cs

```diff
@@ -243,11 +243,18 @@ public RestoreCommand(FrameworkName fallbackFramework)
             var projectLockFilePath = Path.Combine(projectFolder, LockFileFormat.LockFileName);
 
             Runtime.Project project;
-            if (!Runtime.Project.TryGetProject(projectJsonPath, out project))
+            var diagnostics = new List<ICompilationMessage>();
+            if (!Runtime.Project.TryGetProject(projectJsonPath, out project, diagnostics))
             {
                 throw new Exception("TODO: project.json parse error");
             }
 
+            if (diagnostics.HasErrors())
+            {
+                var error = diagnostics.First(x => x.Severity == CompilationMessageSeverity.Error);
```

Comment:
Good point! It's possible that we have more than one unknown framework. I should turn to `AggregateException` here, in order to represent multiple errors. Fixed in follow-up commit.